src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,104 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+type Props = {
+  currentPage: number,
+  itemsToDisplay: Array<string>[],
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination = ({
+  currentPage,
+  itemsToDisplay,
+  onPageChange,
+}: Props) => {
+  const qntyOfCells = getNumbers(1, itemsToDisplay.length);
```

Comment:
It would be more performant to calculate the quantity of pages once when the component mounts or when the total or perPage props change, instead of on every render.